src/data/repositories/utils/DiseaseOutbreakMapper.ts

```diff
@@ -0,0 +1,181 @@
+import {
+    DiseaseOutbreakEventBaseAttrs,
+    HazardType,
+    IncidentStatusType,
+} from "../../../domain/entities/disease-outbreak-event/DiseaseOutbreakEvent";
+import { D2TrackerTrackedEntity, Attribute } from "@eyeseetea/d2-api/api/trackerTrackedEntities";
+import {
+    DiseaseOutbreakCode,
+    DiseaseOutbreakCodes,
+    getValueFromDiseaseOutbreak,
+    isHazardType,
+    isStringInDiseaseOutbreakCodes,
+    KeyCode,
+    RTSL_ZEBRA_ORG_UNIT_ID,
+    RTSL_ZEBRA_PROGRAM_ID,
+    RTSL_ZEBRA_TRACKED_ENTITY_TYPE_ID,
+} from "../consts/DiseaseOutbreakConstants";
+import _c from "../../../domain/entities/generic/Collection";
+import { SelectedPick } from "@eyeseetea/d2-api/api";
+import { D2TrackedEntityAttributeSchema } from "../../../types/d2-api";
+import { D2TrackerEnrollment } from "@eyeseetea/d2-api/api/trackerEnrollments";
+import { getCurrentTimeString } from "./DateTimeHelper";
+
+type D2TrackedEntityAttribute = {
+    trackedEntityAttribute: SelectedPick<
+        D2TrackedEntityAttributeSchema,
+        {
+            id: true;
+            valueType: true;
+            code: true;
+        }
+    >;
+};
+
+function getHazardTypeValue(hazardType: string): HazardType {
+    if (isHazardType(hazardType)) {
+        return hazardType;
+    } else {
+        return "Unknown";
+    }
+}
+
+export function mapTrackedEntityAttributesToDiseaseOutbreak(
+    trackedEntity: D2TrackerTrackedEntity
+): DiseaseOutbreakEventBaseAttrs {
+    if (!trackedEntity.trackedEntity) throw new Error("Tracked entity not found");
+
+    const diseaseOutbreak: DiseaseOutbreakEventBaseAttrs = {
+        id: trackedEntity.trackedEntity,
+        eventId: parseInt(getValueFromMap("eventId", trackedEntity)),
+        name: getValueFromMap("name", trackedEntity),
+        created: trackedEntity.createdAt ? new Date(trackedEntity.createdAt) : new Date(),
+        lastUpdated: trackedEntity.updatedAt ? new Date(trackedEntity.updatedAt) : new Date(),
+        createdByName: undefined,
+        hazardType: getHazardTypeValue(getValueFromMap("hazardType", trackedEntity)),
```

Comment:
This does not seem to work because the hazard types in the trackedEntity are like `BIOLOGICAL_ANIMAL`, `BIOLOGICAL_HUMAN` ... (check https://metadata.eyeseetea.com/rsl-zebra/dhis-web-maintenance/index.html#/edit/otherSection/optionSet/p2N1GkG713S/options) so first we need to map from these codes to  "Biological:Animal",    "Biological:Human",  "Chemical",   "Environmental" or "Unknown". Also there is one new: Biological: Human and Animal